ui/src/Llm.tsx

```diff
@@ -27,6 +28,37 @@ export const rephraseQuestion = async (question: string, conversation: any[], mo
     }
 }
 
+export const get_filters_prompt = (text: string) => {
+    const sys = "Given the following question, extract the value of the following attributes if available. If you cannot confidently extract the value say unknown.\n" +
+        "1. location\n \
+    2. airplane_name\n \
```

Comment:
Is this main-line code or just for NTSB?

Reply:
It's just ntsb for now, will be disabled. Will move it to RPS after